### DISCUSSION TITLE
[BUG] App freezes randomly (repost)

### DISCUSSION THREAD

**Discussion by Soif2Sang** [flet-dev/flet#3385] [Q&A]
### Discussed in https://github.com/flet-dev/flet/discussions/2175

**Description**

Hello there, Few weeks ago I created an issue about this but I was unable to isolate the issue, I created a repository containing just enough code to create my use case.
Very easy, there's few rows, clicks on the "start" buttons that will start fake jobs that only aims to fill the logs and create timers.
Click on the gear, wait few minutes and see that the apps suddenly stops refreshing and NO exception are raised. 
Note: if you don't click on the gear, the application will never freeze.

**Code example to reproduce the issue:**

https://github.com/Soif2Sang/flet_bug

**Describe the results you received:**

The application stops refreshing, page.update() does not work anymore

**Describe the results you expected:**

page.update() to keep working

**Flet version (`pip show flet`)**:

This bug occurs for all flet above 0.7 (I did not test bellow)

**Operating system**:

Windows & mac suffers from this

**Comment by Soif2Sang** [comment]
I reposted because the issue was transfered to discussion which to me is not the right place for bug

**Comment by Soif2Sang** [comment]
```
DEBUG:flet_runtime:Sent: 224
DEBUG:flet_core:_process_command: add [] {'to': '_105', 'at': '220'}
DEBUG:flet_runtime:__send: {"action":"pageControlsBatch","payload":[{"action":"addPageControls","payload":{"controls":[{"t":"text","i":"_731","p":"_105","c":[],"at":"220","value":"[11:10:27] A0fvdB8kgZRq8S5o7N7x","weight":"w600"}],"trimIDs":[]}}]}
DEBUG:flet_runtime:Sent: 224
DEBUG:flet_runtime:Message size: 158
DEBUG:flet_runtime:_on_message: {"action":"pageEventFromWeb","payload":{"eventTarget":"page","eventName":"error","eventData":"RangeError: Invalid value: Not in inclusive range 0..219: 220"}}
DEBUG:flet_core:page.on_event: page error RangeError: Invalid value: Not in inclusive range 0..219: 220
DEBUG:flet_core:_process_command: set ['_12'] {'value': '00:00:21'}
DEBUG:flet_runtime:__send: {"action":"pageControlsBatch","payload":[{"action":"updateControlProps","payload":{"props":[{"i":"_12","value":"00:00:21"}]}}]}
```
I added logging locally and I can see the freeze started right here, when the first error appeared.

**Comment by Soif2Sang** [comment]
@FeodorFitsner can you take a look at it please?

**Comment by FeodorFitsner** [comment]
Honestly, I'm a bit concerned of running a code using shutil and subprocess on my machine. I don't have a sandbox environment though.

I see `"at":"220"` and then `RangeError: Invalid value: Not in inclusive range 0..219: 220`.

What command do you use to run your program?
